Lib/test/test_operator.py

```diff
@@ -290,6 +290,22 @@ def test_rshift(self):
         self.assertEqual(operator.rshift(5, 0), 5)
         self.assertRaises(ValueError, operator.rshift, 2, -1)
 
+    def test_in(self):
+        operator = self.module
+        self.assertRaises(TypeError, operator.in_)
+        self.assertRaises(TypeError, operator.in_, None, None)
+        self.assertRaises(ZeroDivisionError, operator.in_, 1, BadIterable())
+        self.assertTrue(operator.in_(2, range(4)))
+        self.assertFalse(operator.in_(5, range(4)))
```

Comment:
```suggestion
        self.assertTrue(operator.in_(2, [1, 2, 3]))
        self.assertFalse(operator.in_(5, [1, 2, 3]))
```

No need to complicate the tests clarity IMO.

Reply:
This is copy+paste of `contains` test. It is simpler if tests are alike. And I don't want to change `test_contains`.